src/handler.js

```diff
@@ -58,11 +57,11 @@ const getProblemInstance = (event) => {
   return 'queryStringParameters' in event ? `${event.path}?${event.queryStringParameters}` : event.path
 }
 
-const hasQueryParameters = (event) => {
-  return 'queryStringParameters' in event && event.queryStringParameters !== undefined
-}
+const hasQueryParameters = (event) => 'queryStringParameters' in event && event.queryStringParameters !== undefined
```

Comment:
I wonder if we want:

```
const hasQueryParameters = (event) => Object.prototype.hasOwnProperty.call(event, 'queryStringParameters') && (event.queryStringParameters !== null || (Array.isArray(event.queryStringParameters) && event.queryStringParameters.length))

```

This has a couple of benefits, `x in y` interrogates properties of the object and inherited properties. In this case we are only interested in properties of the object. x !== undefined in practice does exactly the same thing. We should also test that the value is not an empty array because a non-array value is not OK. We also need to check that the value is not-null as we are negating the check

The test input needs to be updated for all these possibilities